src/i_system.c

```diff
@@ -44,6 +44,84 @@ ticcmd_t *I_BaseTiccmd(void)
 // I_Error
 //
 
+#ifdef WOOF_DEBUG
+
+#if defined(_WIN32)
+#include <dbghelp.h>
+
+static void PrintBacktrace(void)
+{
+    HANDLE process = GetCurrentProcess();
+
+    boolean symbol_available = false;
+
+    if (SymInitialize(process, NULL, TRUE))
+    {
+        SymSetOptions(SYMOPT_UNDNAME);
+        symbol_available = true;
+    }
+
+    // Capture stack frames
+    void *stack[100];
+    USHORT frames = CaptureStackBackTrace(2, 100, stack, NULL);
+
+    // Print backtrace
+    I_Printf(VB_INFO, "Backtrace (frames %d):", frames);
+    for (USHORT i = 0; i < frames; i++)
+    {
+        DWORD64 address = (DWORD64)stack[i];
+
+        // Buffer for symbol info
+        char buffer[sizeof(SYMBOL_INFO) + 256 * sizeof(char)];
+        PSYMBOL_INFO symbol = (PSYMBOL_INFO)buffer;
+        symbol->SizeOfStruct = sizeof(SYMBOL_INFO);
+        symbol->MaxNameLen = 256;
+
+        // Try to resolve symbol
+        DWORD64 displacement = 0;
+        if (symbol_available
+            && SymFromAddr(process, address, &displacement, symbol))
+        {
+            I_Printf(VB_INFO, " [%2d] %s (0x%llX)", i, symbol->Name, address);
+        }
+        else
+        {
+            I_Printf(VB_INFO, " [%2d] 0x%llX\n", i, address);
+        }
+    }
+}
+
+#else  // POSIX
+
+#include <execinfo.h>
+#include <unistd.h>
+
+static void PrintBacktrace(void)
+{
+    void *stack[100];
+    int size = backtrace(stack, 100);
+    char **symbols = backtrace_symbols(stack, size);
+
+    if (!symbols)
+    {
+        perror("backtrace_symbols");
+        return;
+    }
+
+    // Skip two frames
+    I_Printf(VB_INFO, "Backtrace (frames %d):\n", size - 2);
+    for (int i = 2; i < size; i++)
```

Comment:
Why does it have to start at 2?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/2362/commits/adb5a118ed5d4b12c8db30f5f3622112258e7550